README.md

```diff
@@ -14,60 +21,54 @@ A Drupal Commerce demo store connected with a (testnet) BTCPay Server where you
 * Drupal Commerce 2.x installed ([installation guide](https://docs.drupalcommerce.org/commerce2/developer-guide/install-update/installation))  
 * Drupal: [configured and writable private file system](https://www.drupal.org/docs/8/core/modules/file/overview#content-accessing-private-files)
 
-## Installation and configuration
+## Installation and configuration Guide for BTCPay Server-Drupal Commerce Integration
+
+Ready to accept Bitcoin on your Drupal Store? Follow this quick and easy guide to install and configure the BTCPay Drupal Commerce module. For a quick run through, check out our installation and configuration screencast:
 
-Short walkthrough screencast on installing and configuring the module:
-https://youtu.be/XBZwyC2v48s
+[![BTCPay Server - Drupal Commerce 2.x quick walkthrough](https://img.youtube.com/vi/XBZwyC2v48s/mqdefault.jpg)](https://youtube.com/watch?v=XBZwyC2v48s)
 
-### Quick walkthrough steps
+
+### Easy setup steps
 
 #### Generate pairing code on BTCPay server
-1.  BTCPay server: [create and configure a store](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-btcpay-store)
-2.  in store settings go to "**Access Tokens**"
-3.  click on **[Create a new token]**
+1.  **Setup your store:** You'd need a BTCPay server instance to get started. Don't have one? click [here](https://docs.btcpayserver.org/btcpay-basics/gettingstarted#creating-btcpay-store) for a step by step guide.
```

Comment:
This links to docs how to create a store but you suggest it shows how to create a BTCPay Server instance. Either change text to reflect store or change the link to the deployment options overview.